yt_dlp/extractor/plex.py

```diff
@@ -0,0 +1,471 @@
+import urllib.parse
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    determine_ext,
+    float_or_none,
+    int_or_none,
+    parse_age_limit,
+    traverse_obj,
+)
+
+
+class PlexWatchBaseIE(InfoExtractor):
+    _NETRC_MACHINE = 'plex'
+    # provider url is in https://plex.tv/media/providers?X-Plex-Token=<plex_token>
+    # nb: need Accept: application/json, otherwise return xml
```

Comment:
```suggestion

    # Obtained from https://plex.tv/media/providers?X-Plex-Token=<plex_token>
```
Is this what you meant?

Reply:
yes